Packs/PaloAltoNetworks_IoT3rdParty/Integrations/PaloAltoNetworksIoT3rdParty/PaloAltoNetworksIoT3rdParty.yml

```
@@ -0,0 +1,172 @@
+      name: assetID
+      required: true
+    description: For a given a asset ID (alert-id, vulnerability-id or mac-address)
+      returns the asset details.
+    name: panw-iot-3rd-party-get-single-asset
+    outputs:
+    - contextPath: PanwIot3rdParty.SingleAsset
+      description: Asset Details.
+  - arguments:
+    - auto: PREDEFINED
+      description: Status to be reported to PANW IoT Cloud.
+      name: status
+      predefined:
+      - error
+      - success
+      - disabled
+      required: true
+    - description: Message to be sent to PANW IoT Cloud.
+      name: message
+      required: true
+    - auto: PREDEFINED
+      description: Name of PANW IoT 3rd Party Integration.
+      name: integrationName
+      predefined:
+      - siem
+      - ise
+      - servicenow
+      required: true
+    - description: Name of the playbook.
+      name: playbookName
+      required: true
+    - auto: PREDEFINED
+      description: Type of asset associated with the status report.
+      name: type
+      predefined:
+      - device
+      - alert
+      - vulnerability
+      required: true
+    description: Sends a status message back to PANW IOT cloud.
+    name: panw-iot-3rd-party-report-status-to-panw
+  - arguments:
+    - auto: PREDEFINED
+      description: Input asset type.
+      name: assetType
+      predefined:
+      - Device
+      - Alert
+      - Vulnerability
+      required: true
+    - auto: PREDEFINED
+      description: Desired output format.
+      name: outputFormat
+      predefined:
+      - CiscoISECustomAttributes
+      - SIEM
+      - ServiceNow
+      required: true
+    - description: List of input assets.
+      isArray: true
+      name: assetList
+    - description: 'Example: ServiceNow ID and deviceid mapping.'
+      isArray: true
+      name: metadata
+    - description: 'Example: incident triggered by PANW IoT cloud API.'
+      name: metadata1
+    description: For a given asset (alert, device, vuln) converts it to 3rd party
+      format.
+    name: panw-iot-3rd-party-convert-assets-to-external-format
+    outputs:
+    - contextPath: PanwIot3rdParty.VulnerabilityCEFSyslogs
+      description: List of CEF formatted vulnerability syslogs for SIEM.
+    - contextPath: PanwIot3rdParty.AlertCEFSyslogs
+      description: List of CEF formatted alert syslogs for SIEM.
+    - contextPath: PanwIot3rdParty.DeviceCEFSyslogs
+      description: List CEF formatted device syslogs for SIEM.
+    - contextPath: PanwIot3rdParty.CiscoISEAttributes
+      description: List Cisco ISE attribute dicts/maps.
+    - contextPath: PanwIot3rdParty.AlertServiceNow
+      description: Single SN formatted alert string.
+    - contextPath: PanwIot3rdParty.VulnerabilityServiceNow
+      description: Single SN formatted vulnerability string.
+    - contextPath: PanwIot3rdParty.DeviceServiceNow
+      description: List of upsert ready formatted device for SN.
+  dockerimage: demisto/python3:3.8.6.13358
+  longRunning: true
```

I do not think this is intended
```suggestion
  longRunning: false
```

---

fixed